packages/playground/src/components/vm_deployment_table.vue

```diff
@@ -72,6 +72,7 @@
 
       <template #[`item.planetary`]="{ item }">
         {{ item.value.planetary || "-" }}
+        <v-chip v-if="item.value.mycelium" color="primary" class="text-capitalize"> Mycelium </v-chip>
       </template>
```

Comment:
```suggestion
      <template #[`item.planetary`]="{ item }">
        <div class="d-flex align-center">
          {{ item.value.planetary || "-" }}
          <v-chip v-if="item.value.mycelium" color="primary" class="text-capitalize ml-2"> Mycelium </v-chip>
        </div>
      </template>
```

Currently it looks like that 
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/31689104/894e4aed-24d6-4634-bb44-dbfee044e6a7)